spec/support/feature_helpers.rb

```diff
@@ -322,6 +326,40 @@ def form_is_filled_in_by_form_filler(live_form_link, skip_question: false, confi
     end
   end
 
+  def s3_form_is_filled_in_by_form_filler()
+    s3_form_live_link =  ENV.fetch('S3_FORM_URL') { raise 'You must set $S3_FORM_URL' }
+    form_id =  ENV.fetch('S3_FORM_ID') { raise 'You must set $S3_FORM_URL' }
```

Comment:
I think we could just ask for the `S3_FORM_ID` and not the `S3_FORM_URL` as we can just construct the runner URL from that '#{RUNNER_URL}form/${S3_FORM_ID}' . We would need to start asking for the RUNNER_URL though as doesn't look like we already do that.